src/main/java/com/codecool/virtualpub/logic/Pub.java

```diff
@@ -11,8 +13,10 @@ public class Pub {
     private List<Customer> customers;
     private Bartender bartender;
     private int profit;
+    private Display display = new Display();
+    private Input input = new Input();
```

Comment:
Also, why does Pub need Display and Input classes? If the Game class is our aggregator class, our main logic class, then it should do all the display and input handling. I'm not sure that the Pub needs to be a logic class. It could just store the current game stat as a data class. It would store the drinks and customers and handle them based on what happens in the Game